assets/schema/ensemble_schema.json

```diff
@@ -1569,6 +1569,26 @@
             },
             "trailingText": {
               "type": "string"
+            },
+            "activeColor": {
```

Comment:
Please don't include all properties that don't make sense. Break these up. Each widget should only get properties that apply to them.

Reply:
Can you read the above description of the PR? I just updated